vespalog/src/vespa/log/log.h

```diff
@@ -219,7 +227,7 @@ class Logger {
      *
      * @param timestamp Time in microseconds.
      */
-    void doLogCore(uint64_t timestamp, LogLevel level,
+    void doLogCore(system_time timestamp, LogLevel level,
```

Comment:
Is there any way we can refactor things so that this method doesn't have to have its signature be visible in the main `log.h` file? The `Timer` should be easy to move away and just forward declare. I'm not sure how much heft there is in `<chrono>` but since C++20 adds chrono printing to both both `std::ostream` and `std::formatter` (not yet in GCC's STL), it might grow in transitive size over time. It might be worth reducing the number of transitive includes for a file included by so many others.